build/LocalizationTasks/SaveXlfToResx.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using System.Linq;
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+using xlflib;
+
+namespace Microsoft.Build.LocalizationTasks
+{
+    public class SaveXlfToResx : Task
+    {
+        [Required]
+        public string ResxPath { get; set; }
+
+        [Required]
+        public string XlfPath { get; set; }
+
+        /// <summary>
+        /// When set to "true" it makes the task replace the resource values with another value.
+        /// Useful for testing localized builds.
+        /// 
+        /// If <see cref="TestString"/> is non empty, resource values get replaced with it.
+        /// If it is empty, resource values get replaced with !resource_id!english_resource!localized_resource!
+        /// </summary>
+        public string ReplaceWithTestString { get; set; }
```

Comment:
This seems pretty cool, but is it so strongly coupled with this functionality that it belongs here?